cdf-fabric-replicator-chart/values.yaml

```diff
@@ -0,0 +1,68 @@
+# Default values for cdf-fabric-replicator-chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository:
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
```

Comment:
should we go with help suggested practices for versions: https://helm.sh/docs/chart_best_practices/dependencies/

Reply:
good point removing latest